api.md

```diff
@@ -2654,66 +2656,142 @@ SELECT * FROM timescaledb_information.data_node;
+ public       | dist_table | postgres    |              2 |          3 | f                   | t              |                  2 | {node_1,node_2,node_3}     |
+(1 row)
 ```
 
-Check whether a table is a hypertable.
+## timescaledb_information.dimensions [](timescaledb_information-dimensions)
+
+Get metadata about the dimensions of hypertables.
+The primary dimension of a hypertable is time based.
+The time_interval column is set for hypertables that use date/time datatypes, like timestamp, data etc., for the primary dimension.
+The integer_interval and integer_now_func columns are set for hypertables that use integer datatypes, like bigint, integer etc., for the primary dimension.
+Secondary dimensions are space based and have num_partitions set. The time_interval and integer_interval columns are not applicable to secondary dimensions.
+ 
+#### Available Columns [](timescaledb_information-dimensions-available-columns)
+
+|Name|Description|
+|---|---|
+| `hypertable_schema` | Schema name of the hypertable |
+| `hypertable_name` | Table name of the hypertable |
+| `dimension_number` | Dimension number of the hypertable, starting from 1 |
+| `column_name` | Name of the column used to create this dimension |
+| `column_type` | Type of the column used to create this dimension|
+| `dimension_type` |Is this time based or space based dimension?|
+| `time_interval` | Time interval for primary dimension if the column type is based on Postgres time datatypes |
+| `integer_interval` | Integer interval for primary dimension if the column type is an integer datatype |
+| `integer_now_func` | integer_now function for primary dimension if the column type is integer based datatype|
+| `num_partitions` | Number of partitions for the dimension |
+
+#### Sample Usage [](timescaledb_information-dimensions-examples)
+
+Get information about the dimensions of hypertables.
 
 ```sql
-SELECT * FROM timescaledb_information.hypertable
-WHERE table_schema='public' AND table_name='metrics';
+CREATE TABLE dist_table(time timestamptz, device int, temp float);
+SELECT create_hypertable('dist_table', 'time',  'device', chunk_time_interval=> INTERVAL '7 days', number_partitions=>3);
 
- table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size | distributed
---------------+------------+-------------+----------------+------------+------------+------------+------------+------------+--------------
- public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB     | t
-(1 row)
+SELECT * from timescaledb_information.dimensions
+  ORDER BY hypertable_name, dimension_number;
+
+ hypertable_schema | hypertable_name | dimension_number | column_name |       column_type        | dimension_type | time_interval | integer_interval | integer_now_func | num_partitions
```

Comment:
Maybe switch to extended view `\x` for these longer outputs?

Reply:
fixed.